docs/learn-account-generation.md

```diff
@@ -0,0 +1,241 @@
+
+![subkey-01](assets/accounts/subkey_01.jpg)
+
+## Polkadot.js
+
+Using the polkadot.js user interface without the plugin is not recommended. It is
+the least secure way of generating an account. It should only be used if all of 
+the other methods are not feasible in your situation.
+
+![pjs-01](assets/accounts/polkadot_js_01.jpg)
+
+### Go to Polkadot Apps
+
+Navigate to [Polkadot Apps](https://polkadot.js.org/apps) and click on the "Accounts"
+tab. It is located in the sidebar on the left side of your screen.
+
+### Start Account Generation
+
+Click on the "Add Account" button.
+
+![pjs-02](assets/accounts/polkadot_js_02.jpg)
+
+You should see a pop-up that looks like the one here. The required text fields to complete are highlighted in
+pink.
+
+![pjs-03](assets/accounts/polkadot_js_03.jpg)
+
+You can ignore the Advanced creation options; the defaults are fine. You will have to enter an Account Name
+and a password to protect your account. Be sure to select a secure and hard-to-guess password. Note that
+anything will be accepted as a password here. Please note: There are no checks to see if it is long enough or
+secure. You will need this password for any future interaction with or transaction from this account.
+
+When you have entered valid information for both, the color of the text boxes will turn from pink to white.
+
+![pjs-04](assets/accounts/polkadot_js_04.jpg)
+
+### Store the key somewhere safe
+
+The seed is your **key** to the account. Knowing the seed allows you, or
+anyone else who knows the seed, to re-generate and control this account.
+
+It is imperative to store the seed somewhere safe, secret, and secure. If you lose access to your account, you
+can re-create it by entering the seed. This also means that somebody else can have control over your account
+if they have access to your seed.
+
+For maximum security, the seed should be written down on paper or another non-digital device and stored in a
+safe place. You may also want to protect your seed from physical damage, as well (e.g. by storing in a sealed
+plastic bag to prevent water damage, storing it in a fireproof safe, etc.) It is recommended that you store
+multiple copies of the seed in geographically separate locations (e.g., one in your home safe and one in a
+safety deposit box at your bank).
+
+You should definitely not store your seed on any kind of computer that has or may have access to the internet
+in the future.
+
+### Create and backup account
+
+Click “Save” and your account will be created. It will also generate a backup file that you should save to
+your computer. Ideally, you would also save it on an external hard drive or thumb drive, or print it out and
+be able to re-enter it later. You should not store it in cloud storage, email it to yourself, etc.
+
+You can use this backup file to restore your account. This backup file is not readable unless it is decrypted
+with the password.
+
+![pjs-05](assets/accounts/polkadot_js_05.jpg)
+
+## Coinbase
```

Comment:
Let's remove the Coinbase custody information here.  It's not really specifically about "account generation".